File: rb/test/exists_test.rb
# AntiPhishingDetection SDK exists test

require "minitest/autorun"
require_relative "../AntiPhishingDetection_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = AntiPhishingDetectionSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- AntiPhishingDetection SDK exists test

local sdk = require("anti-phishing-detection_sdk")

describe("AntiPhishingDetectionSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
